marketplace-service/src/main/java/com/axonivy/market/service/MetadataService.java

```diff
@@ -0,0 +1,6 @@
+package com.axonivy.market.service;
+
+public interface MetadataService {
+  void syncAllProductMavenMetadata();
```

Comment:
because is it ALL, so would be nice if we have the 's' e.g: products or medadatas, it's to up